doc/whats-new.rst

```diff
@@ -41,6 +41,9 @@ Bug fixes
 Documentation
 ~~~~~~~~~~~~~
 
+- Update FAQ page on how do I open format X file as an xarray dataset? (:issue:`1285`, :pull:`7638`) using :py:func:`~xarray.open_dataset`
+  By `Harshitha <https://github.com/harshitha1201>`_ , `Tom Nicholas <https://github.com/TomNicholas>`_ and
+  `Michael Niklas <https://github.com/headtr1ck>`_.
```

Comment:
No need to add credits for reviewers, you can remove me from the list :)

Reply:
oh, okay.